src/pycolmap/scene/reconstruction_manager.cc

```diff
@@ -0,0 +1,30 @@
+#include "colmap/scene/reconstruction_manager.h"
+
+#include "pycolmap/pybind11_extension.h"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+using namespace colmap;
+using namespace pybind11::literals;
+namespace py = pybind11;
+
+void BindReconstructionManager(py::module& m) {
+  py::class_<ReconstructionManager, std::shared_ptr<ReconstructionManager>>(
+      m, "ReconstructionManager")
+      .def(py::init<>())
+      .def("size", &ReconstructionManager::Size)
+      .def(
+          "get",
+          [](ReconstructionManager& self, size_t idx) {
+            std::shared_ptr<const Reconstruction> rec = self.Get(idx);
+            return rec;
```

Comment:
Is the lambda needed? Isn't it sufficient to directly bind `ReconstructionManager::Get`? Use `py::overload_cast` to return the non-const version.

Reply:
Done. Thanks